cw/capture.py

```diff
@@ -991,6 +1008,235 @@ def kmac_fvsr_key_batch(ctx: typer.Context,
     capture_end(ctx.obj.cfg)
 
 
+def capture_otbn_vertical(ot, ktp, fw_bin, pll_frequency, capture_cfg):
+    """Capture traces for ECDSA P256/P384 secret key generation.
+
+    Uses a fixed seed and generates several random masks. For the corresponding
+    driver, check:
+    <opentitan_repo_root>/sw/device/sca/ecc256_keygen_serial.c
+
+    Args:
+      ot: Initialized OpenTitan target.
+      ktp: Key and plaintext generator.
+      fw_bin: Firmware binary.
+      pll_frequency: Output frequency of the FPGA PLL.
+      capture_cfg: Capture configuration from the yaml file
+    """
+
+    # We need an intuitive, but non default behavor for the kpt.next() interrator.
+    # For backwards compatibility this must be set in the capture config file.
+    # This is a workaroung for https://github.com/lowRISC/ot-sca/issues/116
+    if "use_fixed_key_iter" not in capture_cfg:
+        raise RuntimeError('use_fixed_key_iter not se set!')
+    if capture_cfg["use_fixed_key_iter"] is not False:
+        raise RuntimeError('use_fixed_key_iter must be set to false!')
+
+    # Currently, we need to reprogram the firmware before every sign operation
+    # TODO: Investigate if this is needed.
+    reset_firmware = False
```

Comment:
Good point. I think @bilgiday investigated it and it is not needed anymore. Should we keep the option?
I'll change the comment, but removing the option should be done in a different PR

Reply:
Your proposal sounds good @m-temp.

Reply:
With the new versions of the device-side firmware, we don't need it. I just kept it here as an option just in case. 

After we merge this PR, I can create a new one to clean `reset_firmware` options from the capture.py. Does that sound good?